README.md

```diff
@@ -158,13 +144,13 @@ let passage_flex = PassageFlex::new(
 let external_id = your_user.id;
 
 // get user info
-let user_info = passage_flex.get_user(external_id).await.unwrap();
+let user_info = passage_flex.user.get_user(external_id).await.unwrap();
```

Comment:
```suggestion
let user_info = passage_flex.user.get(external_id).await.unwrap();
```

Reply:
ah damn it [efd394e](https://github.com/passageidentity/passage-flex-rust/pull/30/commits/efd394e71e4c42a812767c313a0eedc912dc68d3)